scripts/aws/deploy.py

```diff
@@ -47,6 +47,8 @@
                         help="Location of enterprise catalog IDA for request routing")
     parser.add_argument('--authoring-host', required=True,
                         help="Location of Studio for authoring request routing")
+    parser.add_argument('--license-manager-host', required=True,
```

Comment:
For few args like `--enterprise-host` it is marked as `required=False`, how we are figuring this out?